### PR TITLE
Remove brand names from website types

### PR DESCRIPTION
**Pull request by colemanw**
Overview
----------------------------------------
This is a strange list. As far as I can guess, it started reasonably enough with "Work" and "Main" types, then someone added "Facebook" and "MySpace" which are brands, not types (like Kleenex is to tissues). And then it spiraled out of control.

This pulls the list back to being about "types" not brands.

### CODE REVIEW

**Comment by demeritcowboy** [comment]
Some test issues:
CRM_Contact_Import_Parser_ContactTest.testImportLocations
CRM_Core_PseudoConstantTest.testOptionValues

I'm personally ok with it, but it is useful to have them already listed when you're adding them to profiles.

File: sql/civicrm_data/civicrm_option_group/website_type.sqldata.php [hunk 1]
```diff
@@ -6,13 +6,6 @@
   ->addValueTable(['name', 'value'], [
     ['Work', 1, 'is_default' => 1],
     ['Main', 2],
```
Comment (demeritcowboy) [inline sql/civicrm_data/civicrm_option_group/website_type.sqldata.php:8]: Also I think a label param is missing for these first two, and the label would have ts().

Comment (colemanw) [reply sql/civicrm_data/civicrm_option_group/website_type.sqldata.php]: How odd. Ok I've added the ts().

**Comment by demeritcowboy** [comment]
We have one thumb up on the PR, and in chat there's two thumbs up and a "sounds fine". Not super-enthusiasm but enough to merge.

**Pull request merged**